src/sdk/resources/policyTrackingGroups.ts

```diff
@@ -1,10 +1,16 @@
 import { IHttpClient } from '../http';
 import { PagedResource}  from './resource';
 import { PolicyTrackingGroup } from '../models';
+import { CancellationToken } from '../cancellation-token';
 
 export class PolicyTrackingGroupsResource extends PagedResource<PolicyTrackingGroup> {
 	constructor(httpClient: IHttpClient, policyId: string) {
 		super(httpClient, 'tracking_group');
 		this.resourcePath = `policies/${policyId}/tracking_groups`;
 	}
+
+	async clone(id: string, body?: any, params?: any, token?: CancellationToken): Promise<PolicyTrackingGroup> {
```

Comment:
can body parameter be typed? `body? : PolicyTrackingGroup`

Reply:
Yeah good call. Added in new commit.